cmd/celestia/rpc.go

```diff
@@ -219,11 +223,11 @@ func parseParams(method string, params []string) []interface{} {
 
 	for i, param := range params {
 		if param[0] == '{' || param[0] == '[' {
-			var raw json.RawMessage
-			if err := json.Unmarshal([]byte(param), &raw); err == nil {
-				parsedParams[i] = raw
-			} else {
+			rawJSON, err := parseJson(param)
+			if err != nil {
 				parsedParams[i] = param
+			} else {
+				parsedParams[i] = rawJSON
```

Comment:
this would be nicer

```suggestion
                         parsedParams[i] = param
			rawJSON, err := parseJson(param)
			if err == nil {
				parsedParams[i] = rawJSON
			}
```